ui-grid/ui-grid.d.ts

```diff
@@ -490,6 +490,10 @@ declare module uiGrid {
          */
         footerHeight?: number;
         /**
+         * returns or sets grid height in pixels
+         */
+        gridHeight?: number;
```

Comment:
Yes, you are right, I couldn't find it in the doc either. We need it to set grid height so it does not have scrollbars and expands according to its content. That's just a long-lasting ui-grid issue that can be overcome by setting the gridHeight. Good point about using the array accessor, didn't think of that. If it would be bad idea to include this property in the typings, I guess we can live without it.